builtin/logical/database/credentials.go

```diff
@@ -170,3 +175,190 @@ func (kg rsaKeyGenerator) configMap() (map[string]interface{}, error) {
 	}
 	return config, nil
 }
+
+// TODO: subject_template also considered, but we'd need to be
+//       able to parse it from a string -> pkix.Name.
```

Comment:
I think we can remove this todo and leave it posed as a question on the PR. I considered templating of the entire subject but settled for common name since it was more simple.